### ISSUE TITLE
Cannot redeclare exported variable 'ZodSchema'

### ISSUE THREAD

**Issue by dominikzogg** [asteasolutions/zod-to-openapi#68]
I seems that since typescript 4.9.x i get the following

```
node_modules/@asteasolutions/zod-to-openapi/dist/zod-extensions.d.ts:27:20 - error TS2323: Cannot redeclare exported variable 'ZodSchema'.

27     abstract class ZodSchema<Output, Def extends ZodTypeDef, Input = Output> {
```

**Comment by samchungy** [comment]
You're keen 😛 I'm still waiting for typescript-eslint to catchup but I can confirm the above. We could possibly look at converting this to a namespace?

**Comment by AGalabov** [comment]
@samchungy thank you for the PR. @dominikzogg I just released 3.0.1. Please go ahead and try.

**Comment by dominikzogg** [comment]
@AGalabov works.
@samchungy thank you for the fix.
